src/components/back-to-top/back-to-top.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ScrollToTop from "react-scroll-up";
+import { FiArrowUpCircle } from "react-icons/fi";

```

Comment:
this doesn't seem to exist in https://react-icons.github.io
I checked out your branch and i don't see any icon in storybook